pages/pgql-1.1-spec.md

```diff
@@ -1489,6 +1489,21 @@ SELECT p1.name
 
 Above, we compare two string properties from different graphs. Besides properties, it is also possible to compare vertices and edges from different graphs. However, because PGQL 1.1 does not have concepts like graph views, base graphs, or sharing of vertices/edges between graphs, such comparisons will always yield `false`.
 
+## Subqueries Inside PATH Clause
+
+Users can add a sub-query in the `WHERE` clause of the `PATH` definition. One might be intereseted in asserting for specific properties for a node in the `PATH`. The following example defines a path ending in a node which is not the oldest in the graph:
+
+`PATH p AS (a) -> (b) WHERE EXISTS (SELECT * MATCH (x) WHERE x.age > b.age) SELECT ...`
+
+Topology related constraints can be also imposed. The following example defines a path ending in a node which has at least one out-neighbor:
+
+`PATH p AS (a) -> (b) WHERE EXISTS (SELECT * MATCH (b) -> (c)) ...`
+
+Notice that for now only simple constraints (constraints accessing at most one of the pattern variables)
+are allowed under the `WHERE` clause. Queries like:
+
+`PATH p AS (a) -> (b) WHERE EXISTS (SELECT * MATCH (a) -> (c) -> (b)) SELECT ...` are not yet supported.
```

Comment:
Above is just a PGX 2.7 limitation and can't be part of the PGQL 1.1 spec since it is shared by different implementations. You can add it to `pgx/docs/src/reference/pgql-specification.md` though.

Reply:
done